### PR TITLE
Test/120/add fluent validation and write tests

### PR DESCRIPTION
**Pull request by Gorgggulia**
TimelineItemCreateDTO
CreateTimelineItemCommand

FactCreateDTO
CreateFactCommand

Covered validators with tests

CHECK LIST

 СI passed
 Сode coverage >=95%
 PR is reviewed manually again (to make sure you have 100% ready code)
 All reviewers agreed to merge the PR
 I've checked new feature as logged in and logged out user if needed
 PR meets all conventions

### CODE REVIEW

**Review by BlackkTurtle — APPROVED** [review]

**Review by max-khakimzhanov — CHANGES_REQUESTED** [review]
1. What is project.zip file for?
2. Fix build errors

File: Streetcode/Streetcode.BLL/DTO/Timeline/Create/TImelineItemCreateDto.cs [hunk 1]
```diff
@@ -10,5 +11,6 @@ public class TimelineItemCreateDto
         public DateViewPattern DateViewPattern { get; set; }
         public IEnumerable<HistoricalContextDTO>? HistoricalContexts { get; set; } = new List<HistoricalContextDTO>();
         public int StreetcodeId { get; set; }
+        public TeamMemberDTO TeamMember { get; set; }
```
Comment (synapsetron) [inline Streetcode/Streetcode.BLL/DTO/Timeline/Create/TImelineItemCreateDto.cs:14]: Звідки це поле? Його немає в TimelineItem Entity

**Review by synapsetron — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.BLL/MediatR/Timeline/TimelineItem/Create/CreateTimelineItemCommand.cs [hunk 2]
```diff
@@ -7,5 +7,6 @@ namespace Streetcode.BLL.MediatR.Timeline.TimelineItem.Create
 {
     public record CreateTimelineItemCommand(TimelineItemCreateDto timelineItemCreateDto) : IRequest<Result<TimelineItemDTO>>
     {
+     public TimelineItemCreateDto TimelineItem { get; set; }
```
Comment (max-khakimzhanov) [inline Streetcode/Streetcode.BLL/MediatR/Timeline/TimelineItem/Create/CreateTimelineItemCommand.cs:10]: Why did you add this property?
The primary constructor of records type implies that this property is created automatically

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ﻿using FluentResults;
 using MediatR;
+using Streetcode.BLL.DTO.Media.Art;
```
Comment (max-khakimzhanov) [inline Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs:3]: This using directive is not necessary

File: Streetcode/Streetcode.BLL/DTO/Timeline/Create/TImelineItemCreateDto.cs (hunk 1 above)
Comment (max-khakimzhanov) [inline Streetcode/Streetcode.BLL/DTO/Timeline/Create/TImelineItemCreateDto.cs:14]: In the task is not required to change this Dto

**Review by max-khakimzhanov — CHANGES_REQUESTED** [review]
Sonar shows that 2 tests failed

**Comment by vladtymo** [comment]
Too many unnecessary changes. Refactor, clean up the code and create new PR

**Pull request closed without merging**